### PR TITLE
Add family/baby icons/improve metadata

### PR DESCRIPTION
**Pull request by danielbayley**

### CODE REVIEW

File: icons/baby-pacifier.svg
Comment (karsa-mistmere) [inline icons/baby-pacifier.svg:1]: Also kind of looks like a chibi bearded man.

Comment (ygd) [reply icons/baby-pacifier.svg:1]: same perception here.

I suggest
- removing this icon from this PR and continue to have it merged or
- replacing this icon with a standalone pacifier without the baby.

**Comment by ericfennis** [comment]
This PR is merged to the lab branch. This PR does not match our [PR template](https://github.com/lucide-icons/lucide/blob/main/.github/pull_request_template.md), which requires valid use cases. To move on we merged this PR to the [lab branch](https://github.com/lucide-icons/lucide/tree/lab), which will be moved to a separate repo in the future.

**Comment by ygd** [comment]
@ericfennis where can I learn more about: 
> [lab branch](https://github.com/lucide-icons/lucide/tree/lab), which will be moved to a separate repo in the future.

I am interested because thinking about contribution and extensive usage in a new project ...

**Comment by ericfennis** [comment]
@ygd We will start on this hopefully soon. In #1607 are some discussions about this.

**Pull request merged**